pkg/codegen/dotnet/gen_program_expressions.go

```diff
@@ -20,6 +20,8 @@ import (
 	"math/big"
 	"strings"
 
+	"github.com/pulumi/pulumi-java/pkg/codegen/java/names"
```

Comment:
I'd really rather not take a dependency on java from the dotnet codegen. Also, `dotnet` codegen already has this function:
https://github.com/pulumi/pulumi/blob/858f8b20a83eed48649fb833d3659a6873ba508d/pkg/codegen/dotnet/gen.go#L62-L70

Reply:
Copy-paste issue and GoLand imported it automatically. Thanks for the heads-up 🙏